docs/development/writingzeppelininterpreter.md

```diff
@@ -14,7 +14,7 @@ Interpreters in the same InterpreterGroup can reference each other. For example,
 
 <img class="img-responsive" style="width:50%; border: 1px solid #ecf0f1;" height="auto" src="../../assets/themes/zeppelin/img/interpreter.png" />
 
-Interpreter can be launched either using separate classloader or separate JVM process. Sometimes separate classloader causes problem especially when your interpreter uses reflections or trying to grab standard out/err. In this case, separate JVM process is the option you can select. (by checking 'fork' in Interpreter menu, which is default value) When Interpreter is running in separate JVM process, it's communicating with Zeppelin via thrift.
+Interpreter can be launched either using separate JVM process or separate classloader. The default is using separate JVM process (by checking 'fork' in Interpreter menu). When Interpreter is running in separate JVM process, it's communicating with Zeppelin via thrift. The other option is to use separate classloader, note that sometime separate classloader can causes problem especially when your interpreter uses reflections or trying to grab standard out/err.
```

Comment:
With the new interpreter group option, are all the interpreters of the same group running in the same JVM?
I was thinking of the following wording

All Interpreters in the same interpreter group are launched in a single, separate JVM process. The Interpreter  communicate with Zeppelin engine via thrift.